setup/install/providers/aws/aws-ec2.md

```diff
@@ -119,21 +134,24 @@ hal config provider aws account add $AWS_ACCOUNT_NAME \
     --assume-role role/spinnakerManaged
 ```
 
-Now enable AWS
+### Enable the Spinnaker AWS provider
 
 ```bash
 hal config provider aws enable
 ```
 
 ## Advanced account settings
+Once you configure the Spinnaker AWS provider you will be able to manage and deploy EC2 resources with Spinnaker.
 
-You can view the available configuration flags for AWS within the
-[Halyard reference](/reference/halyard/commands#hal-config-provider-aws-account-add).
+You can view the available configuration flags for Spinnaker AWS provider within the
+[Halyard AWS provider reference](/reference/halyard/commands#hal-config-provider-aws-account-add).
 
 ## Next steps
 
-Optionally, you can [set up Amazon's Elastic Container
-Service](/setup/install/providers/aws/aws-ecs/) or [set up another cloud
-provider](/setup/install/providers/), but otherwise you're ready to
-[choose an environment](/setup/install/environment/)
+Optionally, you can enable other AWS Providers:
+[set up Amazon's Elastic Container
```

Comment:
Andres, can you set these as a bulleted list?

Reply:
mmm ok then.